orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/alicloud/AliCloudImageFinder.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2019 Alibaba Group.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.spinnaker.orca.clouddriver.tasks.providers.alicloud;
+
+import static java.util.stream.Collectors.toMap;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.netflix.spinnaker.orca.clouddriver.OortService;
+import com.netflix.spinnaker.orca.clouddriver.tasks.image.ImageFinder;
+import com.netflix.spinnaker.orca.pipeline.model.Stage;
+import java.util.*;
+import java.util.stream.Collectors;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class AliCloudImageFinder implements ImageFinder {
+
+  final String cloudProvider = "alicloud";
+
+  @Autowired OortService oortService;
```

Comment:
we prefer @Autowired constructors over @Autowired fields

Reply:
yourn meacn is 
  private final OortService oortService;

  private final ObjectMapper objectMapper;

  @Autowired
  public OracleImageFinder(OortService oortService, ObjectMapper objectMapper) {
    this.oortService = oortService;
    this.objectMapper = objectMapper;
  }

?

Reply:
yourn mean is
private final OortService oortService;

private final ObjectMapper objectMapper;

@Autowired
public AliCloudImageFinder(OortService oortService, ObjectMapper objectMapper) {
this.oortService = oortService;
this.objectMapper = objectMapper;
}

?